docs/REQUIREMENTS.md

```diff
@@ -16,12 +16,12 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 The following are categorized user stories 
 
 #### Filtering
-1. As a user, I want to query a registry for stored artifacts by digest and tag.
-1. As a user, I want to query the registry for all stored artifacts that reference another artifact by its tag and digest.
-1. As a user, I want to query a registry for all stored artifacts of a particular type that reference another artifact by its tag and digest.
-1. As a user, I want to query a registry for all stored artifacts based on annotations that reference another artifact by its tag and digest.
+1. As a user, I want to query a registry for a stored artifact by its digest or tag.
+1. As a user, I want to query the registry for all stored artifacts that reference a given artifact by its digest or tag.
+1. As a user, I want to query a registry for all stored artifacts of a particular type.
```

Comment:
I like the "that reference another artifact by its tag and digest" qualifier on this, I don't need every sbom in the repository, just the sbom associated with my particular image, but without fetching every attestation and signature.